docs/manage/partitions-data-tiers/index.md

```diff
@@ -27,7 +28,7 @@ You define the data that will reside in a partition by defining a routing expres
 
 ## Limitations 
 
-* To create and manage partitions, you must be an Admin or you must have the Manage Partitions [role capability](../users-roles/roles/role-capabilities.md). 
+* <PartitionPermissions/> 
```

Comment:
For some reason, in this file when the reuse content is placed in a bulleted list, it leaves a large space beneath it. Maybe you should leave it outside the bulleted list to see if it behaves better.

Reply:
good call! I missed this. Just fixed (got rid of the /reuse file for a separate reason).